components/src/preact/MutationAnnotationsContext.tsx

```diff
@@ -67,6 +87,28 @@ function addAnnotationToMap(map: Map<string, MutationAnnotations>, code: string,
 export function useMutationAnnotationsProvider() {
     const mutationAnnotations = useContext(MutationAnnotationsContext);
 
-    return (mutationCode: string, sequenceType: SequenceType) =>
-        mutationAnnotations[sequenceType].get(mutationCode.toUpperCase());
+    return (mutation: Mutation, sequenceType: SequenceType) => {
+        const position =
+            mutation.segment === undefined
+                ? `${mutation.position}`
+                : `${mutation.segment.toUpperCase()}:${mutation.position}`;
+
+        const possiblePositionAnnotations = mutationAnnotations[sequenceType].position.get(position);
+        const possibleExactAnnotations = mutationAnnotations[sequenceType].mutation.get(mutation.code.toUpperCase());
+
+        const annotations =
+            possiblePositionAnnotations && possibleExactAnnotations
+                ? [...possiblePositionAnnotations, ...possibleExactAnnotations]
+                : possiblePositionAnnotations || possibleExactAnnotations;
```

Comment:
Nitpick: I prefer this:
```suggestion
                : (possiblePositionAnnotations ?? possibleExactAnnotations);
```